### PR TITLE
Request to add Marvell FastLinQ QL45000 Series QLogic Corp

### PR DESCRIPTION
**Pull request by hpeaj**
Add HPE Synergy 6820C 25/50Gb Converged Network Adapter - Marvell FastLinQ QL45000
Please refer to the issue 
https://github.com/k8snetworkplumbingwg/sriov-network-operator/issues/234

### CODE REVIEW

File: deploy/configmap.yaml [hunk 1]
```diff
@@ -20,3 +20,4 @@ data:
   Nvidia_mlx5_MT42822_BlueField-2_integrated_ConnectX-6_Dx: "15b3 a2d6 101e"
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
+  Qlogic_qede_QL45000_50G: "1077 1654 1664"
```
Comment (adrianchiris) [inline deploy/configmap.yaml:23]: nit: add newline

File: deployment/sriov-network-operator/templates/configmap.yaml [hunk 2]
```diff
@@ -20,3 +20,4 @@ data:
   Nvidia_mlx5_MT42822_BlueField-2_integrated_ConnectX-6_Dx: "15b3 a2d6 101e"
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
+  Qlogic_qede_QL45000_50G: "1077 1654 1664"
```
Comment (adrianchiris) [inline deployment/sriov-network-operator/templates/configmap.yaml:23]: nit: add newline

File: doc/quickstart.md [hunk 3]
```diff
@@ -17,7 +17,8 @@
 | Mellanox MT28908 Family [ConnectX-6] | 15b3 | 101b |
 | Mellanox MT28908 Family [ConnectX-6 Dx] | 15b3 | 101d |
 | Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx | 15b3 | a2d6 |
-
+| Qlogic  QL45000 Series Gigabit Ethernet Controller | 1077 | 1664 |
```
Comment (adrianchiris) [inline doc/quickstart.md:20]: so are you adding two types of NICs ?

vendor ID : 1077 , device ID 1664
vendor ID: 1077 , device ID 1654 

according to the configMaps above , the PF (physical function) device ID is 1654, and the VF (Virtual function) device ID is 1664

if this is the case, then you need just one line here containing the PF information only.

Comment (hpeaj) [reply doc/quickstart.md]: removed VF ID in readme

File: deployment/sriov-network-operator/templates/configmap.yaml (hunk 2 above)
Comment (hpeaj) [reply deployment/sriov-network-operator/templates/configmap.yaml]: added new line /n

File: deploy/configmap.yaml (hunk 1 above)
Comment (hpeaj) [reply deploy/configmap.yaml]: added new line /n

File: deploy/configmap.yaml [hunk 4]
```diff
@@ -20,4 +20,5 @@ data:
   Nvidia_mlx5_MT42822_BlueField-2_integrated_ConnectX-6_Dx: "15b3 a2d6 101e"
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
-  Qlogic_qede_QL45000_50G: "1077 1654 1664"
\ No newline at end of file
+  Qlogic_qede_QL45000_50G: "1077 1654 1664"
+
```
Comment (adrianchiris) [inline deploy/configmap.yaml:24]: i still see github complaining here :(

File: deployment/sriov-network-operator/templates/configmap.yaml [hunk 5]
```diff
@@ -20,4 +20,5 @@ data:
   Nvidia_mlx5_MT42822_BlueField-2_integrated_ConnectX-6_Dx: "15b3 a2d6 101e"
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
-  Qlogic_qede_QL45000_50G: "1077 1654 1664"
\ No newline at end of file
+  Qlogic_qede_QL45000_50G: "1077 1654 1664"
+
```
Comment (adrianchiris) [inline deployment/sriov-network-operator/templates/configmap.yaml:24]: and here

**Review by adrianchiris — APPROVED** [review]
lets sort out the newline issue, and im LGTM

File: deployment/sriov-network-operator/templates/configmap.yaml (hunk 2 above)
Comment (hpeaj) [reply deployment/sriov-network-operator/templates/configmap.yaml]: image
no matter what i do error remains

File: deploy/configmap.yaml (hunk 1 above)
Comment (hpeaj) [reply deploy/configmap.yaml]: image
no matter what i do error remains

File: deploy/configmap.yaml (hunk 4 above)
Comment (hpeaj) [reply deploy/configmap.yaml]: Any thoughts I have tried multiple combination error still remains ?
https://user-images.githubusercontent.com/55138250/158124068-a51f6746-e9fd-4631-a166-64b5dcea3136.png

File: deployment/sriov-network-operator/templates/configmap.yaml (hunk 5 above)
Comment (hpeaj) [reply deployment/sriov-network-operator/templates/configmap.yaml]: Any thoughts I have tried multiple combination error still remains ?
https://user-images.githubusercontent.com/55138250/158124068-a51f6746-e9fd-4631-a166-64b5dcea3136.png

**Comment by Eoghan1232** [comment]
@adrianchiris @hpeaj 

I fixed the issue with newline.
Please review again.

LGTM

**Review by Eoghan1232 — APPROVED** [review]

**Pull request merged**